templates/indieauth-authorize-form.php

```diff
@@ -33,8 +33,10 @@
 		<?php _e( 'The app is requesting the following <a href="https://indieweb.org/scope">scopes</a>', 'indieauth' ); ?>
 		<ul>
 		<?php
-		foreach ( $scopes as $s ) {
-			printf( '<li><input type="checkbox" name="scope[]" value="%1$s" %2$s /><strong>%1$s</strong> - %3$s</li>', $s, checked( true, true, false ), self::scopes( $s ) );
+		if ( ! empty( $scopes ) ) {
```

Comment:
you should make a func for this that accepts $scopes.

The code at @https://github.com/indieweb/wordpress-indieauth/pull/186/files#diff-d55842c372a35c31dfc5037ff0ffbc250d02cda9da20b6f2edcc4502d0429f60R31-R35 is exactly the same, so pull it out and func it up.